src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -137,6 +208,29 @@ describe('LibraryServiceClient', function() {
         done();
       });
     });
+
+    it('invokes deleteShelf with error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var formattedName = client.shelfPath("[SHELF_ID]");
+      var request = {
+          name : formattedName
+      };
+
+      // Mock Grpc layer
+      client._deleteShelf = function(actualRequest, options, callback) {
+        assert.deepStrictEqual(actualRequest, request);
+        var error = new Error('Fake error');
+        error.code = FAKE_STATUS_CODE;
+        callback(error);
+      };
+
+      client.deleteShelf(request, function(err, response) {
```

Comment:
`deleteShelf` has `Empty` return type, which should make the callback style as `function(err) { ... }`